src/test/watchLater.test.js

```diff
@@ -0,0 +1,24 @@
+import { screen, waitFor } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import { renderWithProviders } from './utils'
+import App from '../App'
+
+it('Watch Later movies page', async () => {
+    renderWithProviders(<App />)
+
+    await userEvent.type(screen.getByTestId('search-movies'), 'forrest gump')
+    await waitFor(() => {
+      expect(screen.getAllByText('Through the Eyes of Forrest Gump')[0]).toBeInTheDocument()
```

Comment:
This way of expecting text could fail due to typos or return multiple movies, use screen.findByText instead.